src/migrations/20221116000000-rtr-goal-performance.js

```diff
@@ -0,0 +1,210 @@
+module.exports = {
```

Comment:
Please add comments to this file. A description of overall function and each section were applicable.